recipes/pcapplusplus/all/conanfile.py

```diff
@@ -34,19 +34,51 @@ def config_options(self):
             del self.options.fPIC
 
     def requirements(self):
-        if self.settings.os != "Windows":
+        if self.settings.os == "Windows":
+            # We don't expose npcap API
+            self.requires("npcap/[>=1.0]")
```

Comment:
Ranged version is not allowed. https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#why-are-version-ranges-not-allowed